File: testing/src/main/java/com/hendraanggrian/material/bannerbar/test/TestActivity.java
package com.hendraanggrian.material.bannerbar.test;

import android.os.Bundle;
import android.widget.FrameLayout;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class TestActivity extends AppCompatActivity {
    public FrameLayout frameLayout;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test);
        setSupportActionBar(findViewById(R.id.toolbar));
        frameLayout = findViewById(R.id.frameLayout);
    }
}

File: testing/src/com/hendraanggrian/material/bannerbar/test/TestActivity.kt
package com.hendraanggrian.material.bannerbar.test

import android.os.Bundle
import android.widget.FrameLayout
import androidx.appcompat.app.AppCompatActivity

open class TestActivity : AppCompatActivity() {
    lateinit var frameLayout: FrameLayout

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)
        setSupportActionBar(findViewById(R.id.toolbar))
        frameLayout = findViewById(R.id.frameLayout)
    }
}